### ISSUE TITLE
Apache module injects redirection code after WordPress causing it to fail

### ISSUE THREAD

**Issue by garrett-eclipse** [johngodley/redirection#629]
### Summary

Using the Apache module the redirection code gets added to the .htaccess file after the WordPress code which causes it to fail and just load the WP 404 page. Editing the .htaccess to move the Redirection code prior to the WordPress block resolves this issue. Please update the Apache module to inject the Redirection code prior to `# BEGIN WordPress`.

### Actual Behavior

Redirection code is injected into the .htaccess after the following WordPress block which causes them to fail. WP Block;
```
# BEGIN WordPress
<IfModule mod_rewrite.c>
RewriteEngine On
RewriteBase /
RewriteRule ^index\.php$ - [L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule . /index.php [L]
</IfModule>

# END WordPress
```

### Steps to reproduce

Create a group using Apache module, set Apache Module setting, create redirects in that group. Try the redirects they will fail, go into .htaccess and move the redirection code before `# BEGIN WordPress` and you'll find they now work.

### Environment

Chrome 63.0.3239.132
WordPress 4.9.2
Redirection 2.10.1
Apache/2.4.29 (cPanel)

**Comment by johngodley** [comment]
This is fixed in the 3.3 beta:

https://github.com/johngodley/redirection/releases/tag/3.3-beta1

**Comment by garrett-eclipse** [comment]
Thanks John, appreciated
